### DISCUSSION TITLE
Help trying to run a nodejs server

### DISCUSSION THREAD

**Discussion by Lmaceira** [deta/cloud-docs#726] [Help]
Hi, I'm new on this, somenone can help me please? 
I'm trying to run a nodejs server on DETA in localhost works great, but when I make de "space push" and try to access in the same way I receive this messages:
In Postman:
![image](https://user-images.githubusercontent.com/28953050/212782026-b9d813b6-e2aa-49a4-adf0-b3240a63c223.png)
In Chrome: 
![image](https://user-images.githubusercontent.com/28953050/212782300-e58b6514-3aa9-4720-959e-cdc3af077d0f.png)

**Comment by sudo-subg** [comment]
Looks like an issue related to Deta's SSL certificate provider. I have pinged the team about this, so you will have a response soon.
